src/app/legacy/containers/Navigation/index.canonical.test.jsx

```diff
@@ -12,6 +13,22 @@ import {
   fireEvent,
 } from '../../../components/react-testing-library-with-providers';
 
+const blocks = [{ id: '1', title: 'Story' }];
+
+const createToggleContextValue = enabled => ({
+  toggleState: {
+    topBarOJs: { enabled },
+  },
+  toggleDispatch: jest.fn(),
+});
+
+const renderWithToggle = (ui, enabled) =>
+  render(
+    <ToggleContext.Provider value={createToggleContextValue(enabled)}>
+      {ui}
+    </ToggleContext.Provider>,
+  );
```

Comment:
Did you know there was a simpler way to render a component with toggles?

See https://github.com/bbc/simorgh/blob/latest/src/app/components/Curation/index.test.tsx#L155-L172 as an example